xikolo-common/ViewHelpers/TableViewResultsControllerDelegateImplementation.swift

```diff
@@ -174,18 +178,35 @@ protocol TableViewResultsControllerConfiguration {
 
     func configureTableCell(_ cell: UITableViewCell, for controller: NSFetchedResultsController<Content>, indexPath: IndexPath)
 
+    func shouldShowHeader() -> Bool
+
+}
+
+extension TableViewResultsControllerConfiguration {
+
+    func shouldShowHeader() -> Bool {
+        return true
+    }
+
```

Comment:
I sometimes wonder wether there is a "protocols" sign in wavy letters above your bed ;P Nice!